### PR TITLE
Add S3/http loader

### PR DESCRIPTION
**Pull request by arthens**
Add `s3_http_loader.py`, which delegates to `s3_loader` or `http_loader` based on the uri scheme.

### CODE REVIEW

File: thumbor_botornado/s3_http_loader.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import thumbor_botornado.s3_loader as S3Loader
+import thumbor.loaders.http_loader as HttpLoader
+import re
+
+def load(context, url, callback):
+	if re.match('https?:', url, re.IGNORECASE):
```
Comment (lwc) [inline thumbor_botornado/s3_http_loader.py]: You could compile this regex for a slight performance gain:
```python
is_http = re.compile('https?:')
if is_http.match(url, re. IGNORECASE):
```
or something like it

**Comment by lwc** [comment]
LGTM :shipit:

**Comment by harto** [comment]
Looks like you accidentally tabs.

**Comment by arthens** [comment]
I didn't add pre-compilation because Python automatically cache the most recent regexps. I guess there's not harm in compiling it tho

https://docs.python.org/2/library/re.html#re.compile

Also anchoring is not required, as `match` only matches from the beginning of the string

https://docs.python.org/2/library/re.html#re.match

**Comment by pda** [comment]
Consider me schooled on python's `re` oddities :)
I guess the benefit of explicitly anchoring is that python-noobs like me will not think it's missing.

**Pull request merged**